src/mongo/db/curop.cpp

```diff
@@ -446,6 +446,8 @@ uint64_t CurOp::MaxTimeTracker::getRemainingMicros() const {
     return _targetEpochMicros - now;
 }
 
+PseudoRandom CurOp::_prng(std::unique_ptr<SecureRandom>(SecureRandom::create())->nextInt64());
```

Comment:
Here SecureRandom is used only to get seed. So its speed is not an issue here. I use this way of initialization because I checked mongo's code base and did not find any case where PseudoRandom is initialized with time(0). Instead I found this way of getting seed from SecureRandom.